dcr/scenarios/extension-telemetry-pipeline/run.py

```diff
@@ -0,0 +1,79 @@
+import glob
```

Comment:
The other scenarios run.py had only having test method calls. so why don't we keep test method calls in separate file to make it consistent across?

Reply:
Everything has function definitions. The test framework just calls `run.host.py` or `run.py`, if you want to call anything else, you need to do it from within these functions. For example The `run.host.py` in `agent-bvt` scenario calls `run1.py` and `run2.py` and executes both of them on the VM itself. These 2 files internally, calls certain functions to test out whatever they want to test out.  

Sample files - 
```py
# run1.py
from dcr.scenario_utils.test_orchestrator import TestFuncObj, TestOrchestrator
from test_agent_basics import test_agent_version, check_hostname, check_ns_lookup, check_root_login

if __name__ == '__main__':
    tests = [
        TestFuncObj("check_agent_version", test_agent_version),
        TestFuncObj("Check hostname", check_hostname),
        TestFuncObj("Check NSLookup", check_ns_lookup),
        TestFuncObj("Check Root Login", check_root_login)
    ]

    test_orchestrator = TestOrchestrator("AgentBVTs-VM", tests=tests)
    test_orchestrator.run_tests()
    test_orchestrator.generate_report_on_vm("test-result-bvt-run1.xml")
    assert not test_orchestrator.failed, f"Test Suite: {test_orchestrator.name} failed"
```

As you can see everything is modularized in this case but at the end of the day its upto the author to however they wanna write it. Like if you would rather call separate files 1 by 1 in the `run.py` then you can definitely do that, that's more implementation specific.

Hope this clears up your doubts. If you need more clarity, feel free to ping me offline and we can hash it out on a call.